lib/src/widgets/ds_bottomsheet_countries.widget.dart

```diff
@@ -0,0 +1,400 @@
+import '../models/ds_country.model.dart';
+import 'fields/ds_app_search_input.widget.dart';
+
+abstract class DSBottomSheetCountries {
+  static final showClearButton = RxBool(false);
+  static final _filterCountries = RxList<DSCountry>([]);
+  static final TextEditingController controller = TextEditingController();
+  static final selectedCountry = Rxn<DSCountry>();
+
+  static show() async {
+    _filterCountries.assignAll(listCountries);
+    return _bottomSheetCountries();
+  }
+
+  static _bottomSheetCountries() {
+    return DSBottomSheetService(
+      fixedHeader: Column(
+        children: [
+          Padding(
+            padding: const EdgeInsets.only(left: 16.0),
+            child: SafeArea(
+              top: false,
+              bottom: false,
+              child: Row(
+                mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                children: [
+                  DSHeadlineLargeText(
+                    'País',
+                  ),
+                  DSIconButton(
+                    onPressed: () {
+                      Get.back();
+                    },
+                    icon: const Icon(DSIcons.close_outline,
+                        color: DSColors.neutralDarkRooftop),
+                  ),
+                ],
+              ),
+            ),
+          ),
+          const DSDivider(),
+          Padding(
+            padding: const EdgeInsets.fromLTRB(
+              16.0,
+              8.0,
+              16.0,
+              8.0,
+            ),
+            child: Obx(
+              () => DSAppSearchInput(
+                color: DSColors.neutralLightSnow,
+                hintText: 'Buscar por nome do país ou código',
+                onSearch: _onSearch,
+                onClear: _onClear,
+                showSuffixIcon: showClearButton.value,
+                controller: controller,
+              ),
+            ),
+          ),
+          const DSDivider(),
+        ],
+      ),
+      context: Get.context!,
+      builder: (_) => _builderCountries(),
+    ).show();
+  }
+
+  static _onSearch(String searchString) {
+    showClearButton.value = searchString.isNotEmpty;
+    _filterCountries.assignAll(
+      listCountries.where((country) =>
+          country.name.toLowerCase().contains(searchString.toLowerCase()) ||
+          country.code.toLowerCase().contains(searchString.toLowerCase())),
+    );
+  }
+
+  static _onClear() {
+    _filterCountries.assignAll(listCountries);
+    controller.clear();
+    showClearButton.value = false;
+  }
+
+  static Widget _builderCountries() {
+    return Obx(
+      () => ListView.builder(
+        itemBuilder: (_, index) {
+          final country = _filterCountries[index];
+          return Obx(
+            () => Padding(
+              padding: const EdgeInsets.only(
+                left: 16.0,
+              ),
+              child: Column(
+                children: [
+                  DSRadioTile<DSCountry>(
```

Comment:
You should do some changes inside ```DSRadioTile``` to achieve a better look by centralizing the title when subtitle is ```null```.

Solution from line 68 to 79:

```dart
Flexible(
              child: Column(
                crossAxisAlignment: CrossAxisAlignment.start,
                children: [
                  title,
                  if (subTitle != null) ...[
                    const SizedBox(
                      height: 5.0,
                    ),
                    subTitle!
                  ],
                ],
              ),
            ),
```

<img width="752" alt="image" src="https://github.com/takenet/blip-ds-flutter/assets/91232521/e9f639fb-7e21-4a69-a9aa-4e13ee2ea0a9">

Reply:
Done.